lua/macro-to-lua.md

```diff
@@ -775,6 +815,40 @@ print('finished waiting')
 
 This provides a function which will be called, which should return true or false, to decide whether to cancel the delay.
 
+> NOTE: `mq.delay` cannot be used within an ImGui callback. It will result in an error, as the ImGui callback is called every frame and cannot be delayed. A required module also cannot include `mq.delay` within its global scope.  
+
+The following examples are invalid uses of `mq.delay`:  

```

Comment:
I'd be careful about including examples of what not to do, they might be taken out of context.

Consider putting this inside of an admonition block? (bit warning block of code or something)
https://squidfunk.github.io/mkdocs-material/reference/admonitions/